src/devicecontroller/NoOpDeviceController.ts

```diff
@@ -69,6 +70,23 @@ export default class NoOpDeviceController
   getVideoInputQualitySettings(): VideoQualitySettings | null {
     return null;
   }
+  getEventController(): EventController {
```

Comment:
To follow the file pattern, could you please add a extra line between variable names and methods?

Reply:
Done!